.github/workflows/jenkins-security-scan.yml

```
@@ -0,0 +1,21 @@
+name: Jenkins Security Scan
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  workflow_dispatch:
+
+permissions:
+  security-events: write
+  contents: read
+  actions: read
+
+jobs:
+  security-scan:
+    uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v2
+    with:
+      java-cache: 'maven' # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
+      # java-version: 21 # Optionally specify what version of Java to set up for the build, or remove to use a recent default.
```

The plugin is currently compiled using `<java.level>8</java.level>` in POM, so perhaps we should set it here as well.

```suggestion
      java-version: 8 # Optionally specify what version of Java to set up for the build, or remove to use a recent default.
```

---

> The plugin is currently compiled using `<java.level>8</java.level>` in POM, so perhaps we should set it here as well.

You are right, at the moment the plugin can not be compiled with Java 17 or 21.
While changing the version for the action would work, I'd propose to upgrade the plugins baseline alongside the required Java version. I'd be happy to provide a PR for that.

---

I see your point, I like your approach better. Now that #76 is merged, we can leave this as you submitted it. I'll solve this conversation.